### DISCUSSION TITLE
Loom style share ui

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#4605] [Ideas]
I shared this replay and loom on the pablo.js hacker news post and the loom shows 15 views and viewer info which is pretty cool. It left me wanting to know the same for my replay. I also like the separate link icon.

Here's what i'd propose, dont show this UI unless another user / even anonymous viewed the replay. If so, show something similar

https://news.ycombinator.com/item?id=29290399
https://www.loom.com/share/059d3d0770db447a8287c05bf269b476
<img width="491" alt="Screen Shot 2021-11-20 at 10 40 06 AM" src="https://user-images.githubusercontent.com/254562/142738030-5ce104c9-909d-4cff-bd2d-a8d35988195a.png">

**Comment by jonbell-lot23** [comment]
I like this a lot. I think it meets a core user need.

**Comment by jonbell-lot23** [comment]
March 15, 2022:

I'm even more excited to do this than I was in November. I don't know that it's necessarily a "core user need" the way I described it above, but it's definitely a helpful bit of polish. We should add it.
